sql/updates/world/2015_09_02_05_world_ahbot_expire_by_house.sql

```diff
@@ -0,0 +1,2 @@
+DELETE FROM `command` WHERE `name` = 'ahbot rebuild';
+INSERT INTO `command` (`name`, `permission`, `help`) values('ahbot rebuild','791','Syntax: .ahbot rebuild [all] [[a]lliance [h]orde [n]eutral]\r\n\r\nExpire all actual auction of ahbot except bided by player. Binded auctions included to expire if \"all\" option used. If one of \"a\" \"h\" or \"n\" option is used, only the auctions of that house will expire otherwise all houses will expire. Ahbot re-fill auctions base at current settings then.');
```

Comment:
I actually missed out on some minor flaws in your SQL when I commented on the missing Line Feed. I also did some spell checking and corrected the grammar of some words. I was uncertain about the exact meaning in the following:
`Expire all actual auction of ahbot except bided by player.` -- bid, bid, bid - just like 'cast', right?
`Binded auctions included to expire if \"all\" option used.`. -- Bind, bound, bound -or- bid, bid, bid?

I have tried correcting them to the following, though I may have missed on the intended meaning:
`Expire all actual auctions of ahbot, except bids by player.`
`Auction bids included to expire if \"all\" option used.` - though I am uncertain if that is what you mean.

Sample with additional grammar corrections:
```sql
DELETE FROM `command` WHERE `name` = 'ahbot rebuild';
INSERT INTO `command` (`name`, `permission`, `help`) VALUES
('ahbot rebuild',791,'Syntax: .ahbot rebuild [all] [[a]lliance [h]orde [n]eutral]\r\n\r\nExpire all actual auctions of ahbot, except bids by player. Auction bids included to expire if \"all\" option used. If one of \"a\" \"h\" or \"n\" option is used, only the auctions of that house will expire. Otherwise, all houses will expire. Ahbot will then refill auctions based on current settings.');
```
Check to see if the intended meanings remain. Remember to add a line break if you copy-paste the above sample. ;)